index.html

```diff
@@ -11,9 +11,223 @@
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
   integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
   crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
   <title>Psych Conf</title>
 </head>
 <body>
-  
+  <main>
+    <header>
+      <div class="topnav" id="myTopnav">
+        <img class="img-responsive logo" 
+        src="images/psych-logo.png" alt="Psych Conference Logo" />
+        <a href="#project">Project</a>
+        <a href="about.html">About</a>
+        <a href="index.html">Home</a>
+        <a href="javascript:void(0);" class="icon menuToggle" id="menuToggle">
+          <i class="fa fa-bars fs-3"></i>
+        </a>
+      </div>
+    </header>
```

Comment:
We need to ensure that your pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

**Navbar**

_Looking at the Design for Mobile Size_

- [ ] The hamburger menu should be on the left side, not on the right side. Also, the logo is hidden.
- [ ] Also, the navbar background is transparent or the same color/pattern as the top section (headline) background.


_The Navbar on Desktop Size:_

- [ ] Is divided into a top dark bar and a bottom white bar.
- [ ] The top short bar section with social media links, logout, and language links aligned to the right.

Please check the images below, observe the differences, and make changes as much as possible.

| Screen Size | Original Design | Your Design |
| ---------------- | -------------------- | ----------------- |
| Desktop | ![original](https://user-images.githubusercontent.com/26856491/124118222-0083f000-da69-11eb-865a-0d1c501b9c79.png) | ![clone](https://user-images.githubusercontent.com/26856491/124118288-1396c000-da69-11eb-94e7-594a0142998e.png) |
| Mobile | ![original](https://user-images.githubusercontent.com/26856491/124118424-3a54f680-da69-11eb-8ced-08a06bdc4b8f.png) | ![clone](https://user-images.githubusercontent.com/26856491/124118435-3f19aa80-da69-11eb-846a-b56e101df9af.png) |

Reply:
Well done working on these requested changes. However, there is still room for improvement. Please see my inline comments under the review 😄